frontend/src/components/reports/CancelSurveyResponseDialog.vue

```diff
@@ -23,36 +23,56 @@ import alertMixin from '@/mixins/alertMixin'
 
 export default {
   name: 'CancelSurveyResponseDialog',
+
```

Comment:
Was there a formatter change here? We don't normally bother with empty lines between declarations.

Reply:
it is updated. sorry it was caused by my inconsistent preference. sometimes I want empty lines between sections, sometimes I don't

Reply:
I hear ya.